float32 pr2_command
float32 roman_command

---

bool result